src/components/MultiHopTrade/components/TradeInput/components/ConfirmSummary.tsx

```diff
@@ -351,13 +340,25 @@ export const ConfirmSummary = ({
               <Text translation='trade.protocolFee' />
             </Row.Label>
             <Row.Value color='text.base'>
-              {protocolFeesParsed?.map(({ amountCryptoPrecision, symbol }) => (
-                <Amount.Crypto
-                  key={`${amountCryptoPrecision}`}
-                  color={redColor}
-                  value={amountCryptoPrecision}
-                  symbol={symbol}
-                />
+              {protocolFeesParsed?.map(({ amountCryptoPrecision, assetId, symbol }) => (
+                <HStack key={`${assetId}-${amountCryptoPrecision}`} justifyContent='flex-end'>
```

Comment:
[non-blocking]: Now that you've guaranteed there is only 1 fee per asset, we can do this probably?

```suggestion
                <HStack key={assetId} justifyContent='flex-end'>
```

Reply:
https://github.com/shapeshift/web/pull/8462/commits/c05650223951f819f01ffd4d055c68d2adf7e212